dd-sdk-android/src/main/kotlin/com/datadog/android/rum/GlobalRumMonitor.kt

```diff
@@ -0,0 +1,136 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-2020 Datadog, Inc.
+ */
+
+package com.datadog.android.rum
+
+import com.datadog.android.core.internal.utils.devLogger
+import com.datadog.android.rum.GlobalRumMonitor.get
+import com.datadog.android.rum.internal.domain.RumContext
+import com.datadog.android.rum.internal.monitor.NoOpRumMonitor
+import java.util.UUID
+import java.util.concurrent.Callable
+import java.util.concurrent.atomic.AtomicBoolean
+import java.util.concurrent.atomic.AtomicReference
+
+/**
+ * A global [RumMonitor] holder, ensuring that all RUM events are registered
+ * on the same instance .
+ *
+ * The [registerIfAbsent] method should only be called once during the application
+ * initialization phase. Any following calls will be no-op. If the [registerIfAbsent]
+ * method is never called, a default no-op implementation is used.
+ *
+ * You can then retrieve the active [RumMonitor] using the [get] method.
+ */
+object GlobalRumMonitor {
```

Comment:
Maybe we should rename this to GlobalRum and rename the `get` method inside to `getMonitor`.  This class has more responsibilities than only handling the current registered monitor.

Reply:
OK